lib/Translation/EmitHLSCpp.cpp

```diff
@@ -692,6 +698,69 @@ void ModuleEmitter::emitConstBuffer(ConstBufferOp op) {
   emitArrayDirectives(op.getResult());
 }
 
+/// Library Ip emitter
+void ModuleEmitter::emitLibraryIp(InstanceOp op) {
+  indent();
+  /// Get Lib name. (This is not used, but left for future updates)
+  // auto libName = op->getAttrOfType<mlir::SymbolRefAttr>("name");
+  // os << libName.getRootReference().getValue().str();
+
+  // Get Ip name, print Ip name.
+  auto ipName = op->getAttrOfType<mlir::SymbolRefAttr>("name");
+  os << ipName.getNestedReferences()[0].getValue().str();
```

Comment:
We probably need to add an API to InstanceOp named getLibraryName()/getIpName() and verify that we only have one entry in getNestedReferences(). We can keep here as it is and update in another patch.